### PR TITLE
fix: DIA-1556: Not getting predictions for all tasks

### PR DESCRIPTION
**Pull request by hakan458**
acks='all' ensures that all replicas have written the message that is produced 

group_id for consumers is to allow multiple consumers to consume from the same topic. In this case we have a 1-to-1 relationship and each group_id should be unique in order to properly track the offset for each topic

manual offset committing in the consumer seems to have solved the issue. 4 in a row runs with 10k/10k outputs. 

We still need the `time.sleep(0.1)` in the submit_batch endpoint, without it we still hit the same offset issue on the consumer side... unclear why exactly.

### CODE REVIEW

**Review by niklub — APPROVED** [review]
Great to see these findings! approved, but please clarify the meaning of each arguments by adding the comments in the code

File: adala/environments/kafka.py [hunk 1]
```diff
@@ -55,14 +55,16 @@ async def initialize(self):
             self.kafka_input_topic,
             bootstrap_servers=self.kafka_bootstrap_servers,
             value_deserializer=lambda v: json.loads(v.decode("utf-8")),
+            enable_auto_commit=False,
```
Comment (matt-bernstein) [inline adala/environments/kafka.py:58]: Should this also be added to the output topic consumer?

Comment (matt-bernstein) [reply adala/environments/kafka.py:58]: also, mind leaving a comment about what this + acks=all does?

Comment (hakan458) [reply adala/environments/kafka.py:58]: Yeah I will add to output as well. For whatever reason, maybe due to rate of messages sent to output topic, we never hit this issue there. Will leave comments as well yes

**Pull request merged**